### DISCUSSION TITLE
Sub Menu Dropdown moved to far left after latest Astroid Framework Update

### DISCUSSION THREAD

**Discussion by southpaw3229** [templaza/astroid-framework#526] [General]
After installing the latest Astroid Framework 3.0.8 , I noticed all my dropdown main menu items are moved to the left and not under the appropriate parent menu item.

The only other update I ran was jollyany-frame work version 3.5.21 which I am also going to bring to their attention.

I am running Joomla 4.4.1 / php 8.2

Not sure if this is a Astroid issue or a Jollyany issue.

See screen shot attached

Thanks

![dropdown-menu-issue](https://github.com/templaza/astroid-framework/assets/6591120/048528bb-1413-449a-9c94-d281ee318cc1)

**Comment by Chacapamac** [comment]
I @southpaw3229 

I think its because 3.0.8 repaired the “**Dropdown Alignment**” for the Sub-Menus container that wasn’t working correctly in 3.0.7.

![sub-menu-alignment](https://github.com/templaza/astroid-framework/assets/1907276/6814169b-bea5-456f-bb98-da01efb8588f)
